.github/workflows/test.yaml

```diff
@@ -49,4 +49,4 @@ jobs:
           
       - name: Test the code
         run: | 
-          protostar test-cairo1
\ No newline at end of file
+          scarb run test
```

Comment:
Should we add a comment here explaining that it's using a script from Scarb.toml?

Reply:
Maybe I will add a line here? https://github.com/maciektr/cairo-template/blob/fb3bf9980d1b605be4781929ba9a2528da549a57/README.md#test

Reply:
Technically, there's a commend about that in Scarb.toml already. But I thought that maybe checking out `tests.yaml`, it's not immediately obvious where does `scarb run test` come from.

Reply:
Hmm, the `scarb run` prefix should point to Scarb scripts anyway, so imho one place pointing this out should be enough.

Reply:
Sounds reasonable, let's just keep it as it is.